apps/web/app/layout.tsx

```diff
@@ -16,8 +16,9 @@ export default function RootLayout({
   return (
     <html lang="ko" className={`${pretendard.variable}`}>
```

Comment:
요거는 지금 봤는데 이렇게 해도 되지 않을까여?
```suggestion
    <html lang="ko" className={pretendard.variable}>
```

Reply:
저렇게 하면 문자열 자체라 리터럴 사용해야 작동합니당